sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_pyamqp_transport.py

```diff
@@ -253,11 +268,20 @@ def create_connection(**kwargs):
         :return: The connection object.
         :rtype: ~pyamqp.Connection
         """
-        endpoint = kwargs.pop("endpoint")
         host = kwargs.pop("host")  # pylint:disable=unused-variable
         auth = kwargs.pop("auth")  # pylint:disable=unused-variable
-        network_trace = kwargs.pop("debug")
-        return Connection(endpoint, network_trace=network_trace, **kwargs)
+        network_trace = debug
+        return Connection(
+            endpoint,
+            container_id=container_id,
+            max_frame_size=max_frame_size,
+            channel_max=channel_max,
+            idle_timeout=idle_timeout,
+            properties=properties,
+            idle_timeout_empty_frame_send_ratio=remote_idle_timeout_empty_frame_send_ratio,
+            network_trace=network_trace,
+            **kwargs
```

Comment:
error_policy, debug, encoding not passed in?

Reply:
`debug` gets set to `network_trace` and that gets passed in as kwarg. The others seemed to be for [`uAMQP.Connection`](https://github.com/Azure/azure-uamqp-python/blob/main/uamqp/connection.py) and not for [`pyamqp.Connection`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_connection.py)

Could still pass them in?